etl/naaccr_etl_postgresql.sql

```diff
@@ -765,11 +774,9 @@ CREATE TABLE naaccr_data_points_temp
     , record_id
   )
 
-  SELECT ( CASE WHEN  (SELECT MAX(condition_occurrence_id) FROM condition_occurrence) IS NULL
-            THEN 0
-            ELSE  (SELECT MAX(condition_occurrence_id) FROM condition_occurrence)
-          END + row_number() over (order by s.record_id)
-      ) AS condition_occurrence_id
+  SELECT COALESCE( (SELECT MAX(condition_occurrence_id) FROM condition_occurrence_temp),
+                 , (SELECT MAX(condition_occurrence_id) FROM condition_occurrence)
```

Comment:
Extra comma at beginning of line.  This is a syntax error.  This error seems to be repeated in every similir context of setting the _id of every temporary table.